packages/core/src/server/index.ts

```diff
@@ -47,65 +51,93 @@ const server = ({ packages }: ServerOptions): ReturnType<Koa["callback"]> => {
   // Setup the settings and store.
   app.use(settingsAndStore(packages));
 
-  // Serve static files.
-  app.use(async (ctx, next) => {
-    const publicPath = ctx.state.store.state.frontity.options.publicPath.replace(
-      /^(?:https?:)?\/\/([^/])+/,
-      ""
-    );
+  app.use((ctx, next) => {
+    // Collect custom middleware.
+    const middleware: Middleware[] = ((server: Package["server"]) => {
```

Comment:
Yeah, you are right. I usually use these functions when there are complex conditions to avoid using `let`, maybe I started with a different approach and then I changed my mind and still left the function here. I'll fix it.